packages/paste-core/components/table/src/types.ts

```diff
@@ -1,6 +1,8 @@
 import * as PropTypes from 'prop-types';
-import {ValueOf} from '@twilio-paste/types';
-import {WidthOptions, isWidthTokenProp} from '@twilio-paste/style-props';
+import type {BoxProps} from '@twilio-paste/box';
```

Comment:
IMHO the `PropType` declarations should be removed since they're not TS types, mixing in the const declarations and the type declarations made this file super confusing to read.

Reply:
I agree, I was confused by this as well. I'm of the mind that each of the PropTypes should be in their respective files instead of this one. We do this in most cases already, and I don't think we're gaining anything from abstracting them to this file, since as Andi said, they're not TS types.

Reply:
For larger components I don't mind them being in a file separate to the component. So for convention I don't mind it on smaller ones too.

I don't like that some time we put prop types into types. I want separate files for both.

Ideal folder strcuture:

```
src/ 
  - index.tsx
  - types.ts
  - proptypes.ts
  - Component.tsx
  - SubComponent.tsx
```

Reply:
I don't mind you pulling them out here in this PR if you want to. It's a small change for this component

Reply:
Okey cool, I'll pull it out into it's own folder. ⛏️